worker-core/src/main/java/com/hpe/caf/worker/core/WorkerCore.java

```diff
@@ -451,27 +456,22 @@ public CoreWorkerCallback(final Codec codec, final ManagedWorkerQueue workerQueu
         }
 
         @Override
-        public void send(final TaskInformation taskInformation, final TaskMessage responseMessage)
+        public void send(final TaskInformation taskInformation, final TaskMessage responseMessage, final boolean publishTaskDataAsObject)
         {
             Objects.requireNonNull(taskInformation);
             Objects.requireNonNull(responseMessage);
             LOG.debug("Sending task {} complete (message id: {})", responseMessage.getTaskId(), taskInformation.getInboundMessageId());
 
             final String queue = responseMessage.getTo();
             checkForTrackingTermination(taskInformation, queue, responseMessage);
-
-            final byte[] output;
-            try {
-                output = codec.serialise(responseMessage);
-            } catch (final CodecException ex) {
-                throw new RuntimeException(ex);
-            }
-
-            final int priority = responseMessage.getPriority() == null ? 0 : responseMessage.getPriority();
-
             try {
+    
+                final byte[] output = convertAndSerializeMessage(responseMessage, publishTaskDataAsObject);
+    
+                final int priority = responseMessage.getPriority() == null ? 0 : responseMessage.getPriority();
+        
```

Comment:
Remove the whitespace from the lines.

```suggestion
                final byte[] output = convertAndSerializeMessage(responseMessage, publishTaskDataAsObject);

                final int priority = responseMessage.getPriority() == null ? 0 : responseMessage.getPriority();

```